src/FAF_wine_mumble_helper

```diff
@@ -0,0 +1,102 @@
+#!/usr/bin/env python2
+
+#-------------------------------------------------------------------------------
+# Copyright (c) 2015 Igor Kotrasinski.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the GNU Public License v3.0
+# which accompanies this distribution, and is available at
+# http://www.gnu.org/licenses/gpl.html
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#-------------------------------------------------------------------------------
+#
+# This script is launched by FAF along with mumble. It listens to the stub
+# mumble_link interface wine-side and forwards calls to linux Mumble.
+#
+# Place it in the same folder as FAForever.exe. Place the mumble_link library
+# in the folder, too.
+
+import socket
+import mumble_link
+
+mumble_sock = None
+client_sock = None
+prefix = ""
+
+class ConnectionEnd(Exception):
+    pass
+
+def link():
+    local = "127.0.0.1"
+    port = 6113
```

Comment:
This is a rather unfortunate choice of local port, being so close to the default that the game itself uses.